py-polars/src/conversion.rs

```diff
@@ -692,6 +692,8 @@ impl<'s> FromPyObject<'s> for Wrap<AnyValue<'s>> {
             Ok(AnyValue::Boolean(ob.extract::<bool>().unwrap()).into())
         } else if let Ok(value) = ob.extract::<i64>() {
             Ok(AnyValue::Int64(value).into())
+        } else if let Ok(value) = ob.extract::<u64>() {
```

Comment:
Can we put this one between `null` and `struct`? This seems a bit rare and will slow down the hotter paths, like float and string.

Reply:
No problem 👍